blackbox/docs/enterprise/monitoring.txt

```diff
@@ -0,0 +1,98 @@
+.. _jmx_monitoring_plugin:
+
+=====================
+JMX Monitoring Plugin
+=====================
+
+The CrateDB monitoring plugin exposes query metrics via the `JMX`_ API.
+
+This plugin is an `enterprise`_ feature.
+
+Enabling the JMX API
+--------------------
+
+.. highlight:: none
+
+To monitor CrateDB using the JMX API, you must to set the following system properties before you start CrateDB::
+
+  com.sun.management.jmxremote
+  com.sun.management.jmxremote.port=<JMX_SERVER_PORT>
+  com.sun.management.jmxremote.ssl=false
+  com.sun.management.jmxremote.authenticate=false
+
+Here, ``<JMX_SERVER_PORT>`` should be replaced with the port number of your JMX server.
+
+More information about the JMX monitoring properties can be found in the `JMX documentation`_.
+
+.. highlight:: sh
+
+You can set system properties with the ``-D`` option::
+
+  sh$ ./bin/crate -Dcom.sun.management.jmxremote \
+  ...             -Dcom.sun.management.jmxremote.port=7979 \
+  ...             -Dcom.sun.management.jmxremote.ssl=false \
+  ...             -Dcom.sun.management.jmxremote.authenticate=false
+
+However, the recommended way to set system properties is via the ``CRATE_JAVA_OPTS`` environment variable, like so::
+
+  sh$ export CRATE_JAVA_OPTS="$CRATE_JAVA_OPTS -Dcom.sun.management.jmxremote ..."
+  sh$ ./bin/crate
+
+Here, replace ``...`` with the rest of the ``-D`` options you would pass to the ``bin/crate`` command on the CLI.
+
+If you're using the CrateDB Debian or RPM packages, you can set this environment variable via the ``/etc/default/crate`` configuration file.
+
+Using Docker
+------------
+
+.. highlight:: none
```

Comment:
can you clarify the last bit, you're saying that the `rmi.port` and `rmi.server.hostname` are both pointing to the Docker instance running the JMX server? and that this should correspond to the address you would use if you were using JConsole?